src/stylesheets/print.scss

```diff
@@ -0,0 +1,29 @@
+@media print {
```

Comment:
Styles for the `print` media query are applied during HTML->PDF conversion by Prince.